tests/test_book.py

```diff
@@ -60,55 +57,52 @@ def test_add_book(view_book, client):
         "Tags ane not the same"
 
 
-def test_add_magazine(view_book, client):
-    view_book.radio.data = 'magazine'
+def test_add_magazine(view_magazine, client):
+    view_magazine.radio.data = 'magazine'
 
     client.post(url_for('library_books.add_book'),
-                data=view_book.data,
+                data=view_magazine.data,
                 follow_redirects=True)
 
-    del view_book.publisher
-    del view_book.original_title
-    del view_book.isbn
-    del view_book.title
-
-    magazine = Magazine.query.filter_by(title=view_book.title_of_magazine.data,
-                                        issue=view_book.issue.data[0]).first()
+    magazine = Magazine.query\
+        .filter_by(title=view_magazine.title_of_magazine.data,
+                   issue=view_magazine.issue.data[0]).first()
     if not magazine:
         assert False, "Data validation failed"
-    assert view_book.title_of_magazine.data == magazine.title, \
+    assert view_magazine.title_of_magazine.data == magazine.title, \
         "The title of the book is not the same as given at the entrance "
-    assert view_book.table_of_contents.data == magazine.table_of_contents, \
+    assert \
+        view_magazine.table_of_contents.data == magazine.table_of_contents, \
```

Comment:
This is more characters then stickler allows?

Reply:
Could be written just like this:
```suggestion
    assert view_magazine.table_of_contents.data == magazine.table_of_contents,\
        "The table of content is not the same as given at the entrance "
```
It passes flake8 checks on my machine